dask_kubernetes/operator/deployment/Dockerfile

```diff
@@ -10,13 +10,5 @@ WORKDIR /src/dask_kubernetes
 # Install dependencies
 RUN pip install .
 
-# We don't use TARGETARCH so as to support non-buildkit builds
-RUN MAGICARCH=$(dpkg --print-architecture) && \
-    curl -LO "https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/${MAGICARCH}/kubectl" && \
-    mkdir -p /usr/local/bin && \
-    mv ./kubectl /usr/local/bin/kubectl && \
-    chmod +x /usr/local/bin/kubectl && \
-    kubectl version --client
-
```

Comment:
The cluster manager still uses `kubectl` in a few of the methods.

Reply:
The cluster manager will not be used within the operator container though right? Just on the user's machine.

Reply:
Yes, that's right.